### DISCUSSION TITLE
一个AI翻译的小问题

### DISCUSSION THREAD

**Discussion by TNCNET** [share121/inter-knot#5311] [灌水]
之前在尝试AI翻译的时候，遇到这么个问题

原文：“欢迎新老师生前来就餐”
通义千问：
"Welcome new teachers and students for dining"

Deepl：
“Welcome to the New Teacher's Birthday Dining”

而ai的翻译老是忽略了“old”这一部分（恼）

我自己可能会写成 welcome all student and teacher（s） for dine.

这可能是分析得到的结论么？或者是别的？

**Comment by IXfromThorne** [comment]
某些AI不会断句，确信。
人：新老（即“全体”），师生
通义千问：新 老师 （学）生
Deepl……给我整不会了

**Reply by TNCNET** [reply]
实际上，这两个例子都没出现“old”这个概念--但偶然间发现，其他一些人的翻译结论是基本正确的。不过你说的也对：

AI的逻辑下， 可能是新/老师/学生；
人的话，可能会是「新老/师生」。
